stix_shifter_modules/datadog/configuration/config.json

```diff
@@ -7,6 +7,10 @@
             "type": "text",
             "regex": "https?:\\/\\/((app|us3|us5)\\.)[a-z0-9-]+\\.[a-z]{2,3}$"
         },
+        "selfSignedCert": {
```

Comment:
You will also need to have an entry in the `lang_en.json` file. Example
```
"selfSignedCert": {
            "label": "Datadog Certificate",
            "placeholder": "Paste your certificate"
        }
```

Reply:
Added certificate entry in lang_en.json